server/api/collections/index.get.ts

```diff
@@ -0,0 +1,42 @@
+import { getDatabaseConnection } from "@/server/database/dbConnection";
```

Comment:
There is a very high amount of duplicity between the GET and POST endpoint code for `collections` and `incidents`. 

Could we consider either doing a single collections endpoint with a `type` query param e.g. `/api/collections?type=incident`, or at least some shared handler functions?

Reply:
I think the truth is we probably don't yet whether we will forever want separate endpoints `/api/collections` + `/api/incidents` or something more dynamic. 

If I were in your shoes, I'd probably lean toward keeping separate endpoints.  And then the internal implementation of these can drift into shared handler functions as we understand more about what is vs is not duplicated.